docs/processors.md

```diff
@@ -120,6 +127,9 @@ namespace App\BoardGameBlog\Infrastructure\Sylius\Resource;
 final class BoardGameResource implements ResourceInterface
 ```
 
+Note that in that kind of operation, you can disable providing data.
```

Comment:
> Note that in that kind of operation, you can disable providing data.

What kind of operation? Would be cool to re-phrase a bit to be more specific

Reply:
Once I've read "Disable providing data" section, I've understood the point (delete, put, patch), but `note` phrasing needs to reflect it somehow.

Reply:
I've replaced "in that kind of operation" with "in a delete operation".